### ISSUE TITLE
About the intents of queries

### ISSUE THREAD

**Issue by gao-xiao-bai** [microsoft/MIMICS#2]
In your paper "Analyzing and Learning from User Interactions for Search Clarification", you use two datasets for estimating the
intents of each query. However, you did not release the intent data. Will you release the intent data to promote the research on MIMICS?

**Comment by hamed-zamani** [comment]
No, that was obtained from session data which is not part of the release plan due to the sensitive information they may contain.

**Comment by gao-xiao-bai** [comment]
OK. Thank you.
